src/controllers/TalksController.php

```diff
@@ -124,9 +124,14 @@ public function postAction($request, $db)
                         $comment    = $comment_mapper->getCommentById($new_id);
                         $speakers   = $talk_mapper->getSpeakerEmailsByTalkId($talk_id);
                         $recipients = array();
+                        $commentAuthorEmail = $comment[0]['email'];
```

Comment:
Is this additional assignment necessary? As you're not calling a function there it should be possible to check against ```$comment[0]['email']``` on every foreach loop.